mesonbuild/compilers/compilers.py

```diff
@@ -1082,6 +1118,9 @@ def get_link_whole_for(self, args):
     def get_instruction_set_args(self, instruction_set):
         return gnulike_instruction_set_args.get(instruction_set, None)
 
+    def get_default_include_dirs(self):
+        return gnulike_default_include_dirs(self.exelist, self.language)
```

Comment:
This should also be copied into `IntelCompiler` since it has the same syntax as gcc and clang. Does it also make sense to implement this for MSVC? That would involve just reading the `INCLUDE` env var, I expect.

Reply:
Ok